its/src/test/java/com/sonar/it/csharp/MetricsIncludeHeaderCommentTest.java

```diff
@@ -53,40 +57,10 @@
   private static final String FILE = "MetricsTest:foo/Class1.cs";
   private static final int NUMBER_OF_HEADER_COMMENT_LINES = 2;
 
+  // This is where the analysis is done.
   @ClassRule
   public static RuleChain chain = getRuleChain();
 
-  private static RuleChain getRuleChain() {
```

Comment:
I did this change in every file, just moved the private static method at the end